### PR TITLE
MGMT-10891: Rephrase Custom Domain page

### PR DESCRIPTION
**Pull request by mareklibra**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainPage.test.tsx.snap [hunk 1]
```diff
@@ -338,7 +338,7 @@ exports[`DomainPage can render 1`] = `
               >
                 Optional
               </span>
-               Create unique URLs for your edge cluster, such as device setup and console.
+               Enter the name of root domain for your edge cluster (e.g. yourdomain.com)
```
Comment (flaper87) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainPage.test.tsx.snap:341]:
```suggestion
               Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

**Review by flaper87 — CHANGES_REQUESTED** [review]
small wording nit, otherwise LGTM

(let's change all the `root` domain wth `base domain`, which aligns better with OCP terminology

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 2]
```diff
@@ -25,7 +25,7 @@ exports[`DomainSelector can can handle invalid domain 1`] = `
       >
         Optional
       </span>
-       Create unique URLs for your edge cluster, such as device setup and console.
+       Enter the name of root domain for your edge cluster (e.g. yourdomain.com)
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:28]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 3]
```diff
@@ -110,7 +110,7 @@ exports[`DomainSelector can can handle valid domain 1`] = `
...
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:113]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 4]
```diff
@@ -190,7 +190,7 @@ exports[`DomainSelector can render 1`] = `
...
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:193]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainPage.test.tsx.snap [hunk 5]
```diff
@@ -338,7 +338,7 @@ exports[`DomainPage can render 1`] = `
               >
                 Optional
               </span>
-               Create unique URLs for your edge cluster, such as device setup and console.
+               Enter the name of baseu domain for your edge cluster (e.g. yourdomain.com)
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainPage.test.tsx.snap:341]: mistype
```suggestion
               Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 6]
```diff
@@ -25,7 +25,7 @@ exports[`DomainSelector can can handle invalid domain 1`] = `
       >
         Optional
       </span>
-       Create unique URLs for your edge cluster, such as device setup and console.
+       Enter the name of baseu domain for your edge cluster (e.g. yourdomain.com)
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:28]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 7]
```diff
@@ -110,7 +110,7 @@ exports[`DomainSelector can can handle valid domain 1`] = `
...
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:113]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap [hunk 8]
```diff
@@ -190,7 +190,7 @@ exports[`DomainSelector can render 1`] = `
...
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/__snapshots__/DomainSelector.test.tsx.snap:193]:
```suggestion
       Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

File: ui/frontend/src/components/DomainPage/DomainSelector.tsx [hunk 9]
```diff
@@ -17,8 +17,7 @@ export const DomainSelector: React.FC = () => {
         <Title headingLevel="h1">Do you want to use a custom domain?</Title>
       </StackItem>
       <StackItem className="wizard-sublabel">
-        <OptionalBadge /> Create unique URLs for your edge cluster, such as device setup and
-        console.
+        <OptionalBadge /> Enter the name of baseu domain for your edge cluster (e.g. yourdomain.com)
```
Comment (eifrach) [inline ui/frontend/src/components/DomainPage/DomainSelector.tsx:20]:
```suggestion
        <OptionalBadge /> Enter the name of base domain for your edge cluster (e.g. yourdomain.com)
```

**Review by eifrach — CHANGES_REQUESTED** [review]

**Review by eifrach — APPROVED** [review]

File: ui/frontend/src/components/DomainPage/DomainSelector.tsx (hunk 9 above)
Comment (mareklibra) [reply ui/frontend/src/components/DomainPage/DomainSelector.tsx]: Ooops, thanks.

**Review by alknopfler — COMMENTED** [review]
LGTM

**Pull request merged**